framework/tester.py

```diff
@@ -359,3 +359,17 @@ def wait_all_connections(self, tmt=1):
             if not srv.wait_for_connections(timeout=tmt):
                 return False
         return True
+
+    def start_all_services(self, deproxy: bool = False) -> None:
+        """
+        Start all services.
+
+        Args:
+            deproxy (bool): True - if server is deproxy
+        """
+        self.start_tempesta()
+        self.start_all_servers()
+        self.start_all_clients()
+        if deproxy:
```

Comment:
What about running deproxy manager if Deproxy is present in the list of backends?
Perhaps this will help to avoid errors with the flag.

Reply:
Yes, I agree.